engine/src/main/java/engine/physic/PhysicManager.java

```diff
@@ -16,10 +17,19 @@
     private final GameField field;
     private final Collection<GameObject> gameObjects;
     private Set<Interaction> interactions;
+    private final double frictionCoefficient;
 
     public PhysicManager(GameField field) {
         this.field = field;
         this.gameObjects = new HashSet<>();
+        this.frictionCoefficient = 0.1;
```

Comment:
Lets move this coefficient as private static final double FRICTION_COEFFICIENT to GameField